.platform/services.yaml

```diff
@@ -1,3 +1,6 @@
 mysql:
```

Comment:
You may also want to rename this service to mysqldb or similar, to avoid confusion.